src/components/App.js

```diff
@@ -17,14 +19,35 @@ class App extends Component {
     const questionsUrl = () => `http://localhost:3001/quizzes`;
     return fetch(questionsUrl())
     .then(result => result.json())
-    .then(jsonResult => console.log(jsonResult))
+    .then(result => {this.setState({
+      quizzes: result.quizzes,
+      questions: result.quizzes[0].questions,
+    });
+      console.log(result);
+    })
     .catch(error => console.log(error));
   }
 
   render() {
     return (
       <div className="App">
-        <h1>What JavaScript Error Are You?</h1>
+        {this.state.quizzes.map(quiz => { return (
+            <div key={quiz.id}>
+              <h1>{quiz.title}</h1>
+            </div>
+          )}
+        )}
+        {this.state.questions.map(question => { return (
```

Comment:
We might want to break up this block of HTML we're returning by creating some additional components. If we add a Question component and an Answer component, we could render something like the following:

```javascript
questions.map((question, index) =>
    <Question title={question.title} key={question.id}/>
    (question.answers).map(answer =>
        <Answer title={answer.title} />
    )
```

(That is not going to be the exact syntax, and I'm sure it will break in all sorts of ways, but that's what we should be going for here.)